### ISSUE TITLE
Toggle incorrectly states role is "button" and doesn't read on/off state in Chrome based browsers

### ISSUE THREAD

**Issue by dPJinjo** [OfficeDev/office-ui-fabric-react#11053]
### Environment Information

- **Package version(s)**: 7.56.3
- **Browser and OS versions**: Edge 79.0.309.7 (and Chrome) + Narrator; Windows 18363.418

### Describe the issue:

Toggle incorrectly states role is "button" and doesn't read on/off state in Chrome based browsers. Does appear to work correctly in Edge v 44.18362.387.0. Observing UI tree with browser dev tools, it appears like all the correct aria tags have sane values. It's not clear to me why Narrator isn't speaking the values provided.

### Please provide a reproduction of the issue in a codepen:

I can't seem to get Toggle to work in codepen at the moment, but issue is observable on documentation site https://developer.microsoft.com/en-us/fabric#/controls/web/toggle

#### Actual behavior:

Upon tab focus of a toggle, Narrator reads "-label- button".

#### Expected behavior:

Should read something like "-label- switch -onText|offText-".

**Comment by maxwellred** [comment]
Thank you @dPJinjo ! I'll look into this and get back to you.

**Comment by maxwellred** [comment]
Hi @dPJinjo I have tested these controls and haven't be able to reproduce the Windows + Narrator combination (as in it reads them correctly as = "-label- toggle switch on/off"). However, I have observed this issue in the latest build of Chrome. Could you please re-test in the Windows + Narrator and confirm as an issue with Chrome + Narrator would be an issue to file with either the Narrator team or Chrome. Also I noticed your Windows OS build is 1836**3** (ver.1909), I have only been able to update to 18362 (ver. 1903) and can't see your version in the [official release info](https://docs.microsoft.com/en-us/windows/release-information/). Do you happen to know if they rolled back the update? Thanks!

**Comment by dPJinjo** [comment]
![image](https://user-images.githubusercontent.com/16249087/68969495-f466f400-0799-11ea-9c12-393e93b9cce5.png)

![image](https://user-images.githubusercontent.com/16249087/68969542-0f396880-079a-11ea-9c0e-0e71c34246a9.png)

Repro Screen Recording
[ToggleButton.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/3852885/ToggleButton.zip)

**Comment by maxwellred** [comment]
Hello @dPJinjo , I have been able to finally download the latest OS 1909 and have tested the code Windows + Edge and it works as intended. I do see the Issue repro-ing with Edge insider versions(Beta/Dev) but suggested by the a11y team to search "edge://flags/" in Edge (and Beta/Dev/Can), search "ui automation" > select "Native UI Automation" > set to "Enabled" > press Reset. After doing these steps Narrator read the text correctly in all version. As for Chrome, I was told that the a11y API used by that browser bridges poorly to the API Narrator/Windows consumes and will yield poor UX. They are currently working on this issue and many more but I'd recommend using the Edge + Narrator pairing for the best Narrator experience.

**Comment by dPJinjo** [comment]
I am using Edge + Narrator, as the bug describes. It wasn't the best experience. Do you perhaps mean that the insider version of Edge isn't fully supported yet?

I was able to confirm that using the flag you pointed out that I am getting a better Narrator experience. It does narrate the toggle states and announces state changes when the Toggle has focus. Thanks for that.
